src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -1596,3 +1618,123 @@ uploadReachabilityContent signedUrl bs = fossaReq $ do
         lbsResponse
         options
         (pure . requestEncoder)
+
+policiesURLEndpoint :: Url 'Https -> Url 'Https
+policiesURLEndpoint baseUrl = baseUrl /: "api" /: "policies"
+
+getPolicies ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has Debug sig m
+  ) =>
+  ApiOpts ->
+  m [Fossa.API.Types.Policy]
+getPolicies apiOpts = fossaReq $ do
+  (baseUrl, baseOpts) <- useApiOpts apiOpts
+  resp <- req GET (policiesURLEndpoint baseUrl) NoReqBody jsonResponse baseOpts
+  pure (responseBody resp)
+
+teamsURLEndpoint :: Url 'Https -> Url 'Https
+teamsURLEndpoint baseUrl = baseUrl /: "api" /: "teams"
+
+getTeams ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has Debug sig m
+  ) =>
+  ApiOpts ->
+  m [Fossa.API.Types.Team]
+getTeams apiOpts = fossaReq $ do
+  (baseUrl, baseOpts) <- useApiOpts apiOpts
+  resp <- req GET (teamsURLEndpoint baseUrl) NoReqBody jsonResponse baseOpts
+  pure (responseBody resp)
+
+deleteReleaseGroupURLEndpoint :: Url 'Https -> Text -> Url 'Https
+deleteReleaseGroupURLEndpoint baseUrl releaseGroupId = baseUrl /: "api" /: "project_group" /: releaseGroupId
+
+deleteReleaseGroup ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has Debug sig m
+  ) =>
+  ApiOpts ->
+  Text ->
+  m ()
+deleteReleaseGroup apiOpts releaseGroupId = fossaReq $ do
+  (baseUrl, baseOpts) <- useApiOpts apiOpts
+  void $ req DELETE (deleteReleaseGroupURLEndpoint baseUrl releaseGroupId) NoReqBody ignoreResponse baseOpts
+
+releaseGroupReleaseURLEndpoint :: Url 'Https -> Text -> Text -> Url 'Https
+releaseGroupReleaseURLEndpoint baseUrl releaseGroupId releaseId = baseUrl /: "api" /: "project_group" /: releaseGroupId /: "release" /: releaseId
+
+deleteReleaseGroupRelease ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has Debug sig m
+  ) =>
+  ApiOpts ->
+  Text ->
+  Text ->
+  m ()
+deleteReleaseGroupRelease apiOpts releaseGroupId releaseId = fossaReq $ do
+  (baseUrl, baseOpts) <- useApiOpts apiOpts
+  void $ req DELETE (releaseGroupReleaseURLEndpoint baseUrl releaseGroupId releaseId) NoReqBody ignoreResponse baseOpts
```

Comment:
[blocker] use `context` for explainability, and debugging. This applies to all requests here

```
response <-
    context ("Retrieving a signed S3 URL for reachability content upload") $
      req POST (signedReachabilityContentURLEndpoint baseUrl .....
```